srcpkgs/qodem/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'qodem'
+pkgname=qodem
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+make_build_args="-f ./build/Makefile.generic"
+make_build_target="clean qodem"
+makedepends="ncurses-devel gpm-devel"
+short_desc="Qodem is a re-implementation of the Qmodem serial communication package"
```

Comment:
s/Qodem is a r/R/

Reply:
> s/Qodem is a r/R/

Sorry, I'm not sure what you mean by this

Reply:
remove the "Qodem is a" from the line (it's written as a sed substitution)

Reply:
Sorry, right.  It should read as "Re-implementation of the Qmodem serial communication package".